README.md

```diff
@@ -613,7 +613,31 @@ Snapshots of the development version are available in [Sonatype's `snapshots` re
 R8 / ProGuard
 --------
 
-Moshi contains minimally required rules for its own internals to work without need for consumers to embed their own. However if you are using reflective serialization and R8 or ProGuard, you must add keep rules in your proguard configuration file for your reflectively serialized classes.
+If using Android, the Android Gradle Plugin should automatically extract embedded rules included in 
```

Comment:
One open bit from the previous PR: https://github.com/square/moshi/pull/1076#commitcomment-37485365